src/components/Plan.js

```diff
@@ -38,6 +41,41 @@ class Plan extends Component {
     }
   }
 
+  onFavPress(station){
```

Comment:
This logic to store a single station is duplicated in ‘Plan.js‘ and 'StationCard.js'.
Would be nice to extract it to the wrapper mentioned in other comment where we could have a method e.g. like
´addToFavourites (station)´